index.js

```diff
@@ -225,6 +225,10 @@ alexa.request = function(json) {
     }
     return this.data.request.type;
   };
+  this.isInSkillPurchase = function() {
+    var requestType = this.type();
+    return (requestType && 0 === requestType.indexOf("Connections."));
+  };
```

Comment:
Can you explain how "Connections." means "isInSkillPurchase"? Point to an Amazon doc?

Reply:
If we're going to handle these specifically I think the check should be checking either for "Connections.SendRequest" or "Connections.Response", before we get "Connections.DoSomethingElse" that means ... something else. But see my bigger questions below.